db/schema.rb

```diff
@@ -1065,6 +1065,14 @@
     t.string "var", null: false
     t.index ["var"], name: "index_settings_campaigns_on_var", unique: true
   end
+  
```

Comment:
for some weird reason I get this diff when I run `db:migrate` locally:

```diff
diff --git a/db/schema.rb b/db/schema.rb
index 2fa5d86a8..6382f97b9 100644
--- a/db/schema.rb
+++ b/db/schema.rb
@@ -1065,7 +1065,7 @@
     t.string "var", null: false
     t.index ["var"], name: "index_settings_campaigns_on_var", unique: true
   end
-
+
   create_table "settings_communities", force: :cascade do |t|
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
```

Reply:
I had resolved a conflict directly in Github, apparently the Enter key auto-indents but there's no visible whitespace, something I always turn on in my editors.